src/models/accounts/checking-account.model.ts

```diff
@@ -0,0 +1,46 @@
+import { AccountType } from 'src/enums/type-account.enum';
+import { Account } from './account.model';
+
+export class CheckingAccount extends Account {
+  public overdraft: number = 100.0;
```

Comment:
Gostei também que você evitou o uso de números mágicos. Se de fato `overdraft` for uma constante, você pode usar a palavra reservada `readonly`